install/kubernetes/cilium/templates/cilium-agent/service.yaml

```diff
@@ -44,3 +45,24 @@ spec:
     targetPort: envoy-metrics
 {{- end }}
 {{- end }}
+{{- if and .Values.hubble.enabled (hasKey .Values.hubble "listenPort") (hasKey .Values.hubble "peerServicePort") }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: hubble-peer
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: cilium
+spec:
+  selector:
+    k8s-app: cilium
+  ports:
+  - name: peer-service
+    port: {{ .Values.hubble.peerServicePort }}
```

Comment:
Can we have the same logic as for [Hubble Relay](https://github.com/cilium/cilium/blob/8302f0ec299a3f9100ac72d2d0c113466de9894a/install/kubernetes/cilium/values.yaml#L779-L782) here?

Reply:
Can you elaborate?

Reply:
When the Helm value for the svc port is not defined, Relay uses 80 or 443 when server side TLS is disabled or enabled, respectively. See the [Helm value doc](https://github.com/cilium/cilium/blob/f0fbba16a07d84d0ffc8c64bac3b5442e49a37ea/install/kubernetes/cilium/values.yaml#L779-L782) and [svc port logic](https://github.com/cilium/cilium/blob/8302f0ec299a3f9100ac72d2d0c113466de9894a/install/kubernetes/cilium/templates/hubble-relay/service.yaml#L15-L19).

Reply:
Oh you want the peer service on by default? Then should we just get rid of the [socket mount](https://github.com/cilium/cilium/pull/18620/files#diff-d53a1231b790c002402794843350b2e49a649a5cc325a35655793e89222bc808R78-R82) altogether in hubble-relay?

Reply:
>  Oh you want the peer service on by default? Then should we just get rid of the socket mount altogether in hubble-relay?

To answer the questions: yes I think the peer svc should be the default, and we should get rid of the socket mount (maybe deprecated it first). I don't see the point to keep the socket mount around once we have a working peer svc over TCP.